### DISCUSSION TITLE
Govee Wi-Fi Digital Thermometer Hygrometer

### DISCUSSION THREAD

**Discussion by ttais2017** [Joery-M/Govee-LAN-Control#7] [General]
Hi Joery, 
Thanks for sharing your work.  ;) 
I would like to ask if the GoveeLANControl can also be used for getting measurements from Thermometer Devices?  If you guess it could be used for that, I will start with testing.
Best Regards

**Comment by Joery-M** [comment]
Unfortunately Govee only supports (some) lights with their LAN API, but if possible you can check out [this project](https://github.com/home-is-where-you-hang-your-hack/sensor.goveetemp_bt_hci), it uses Bluetooth.

Also, if Govee were to support thermometers, I would have to own one, which I don't. (After the issues I've been having with my single light, I can't recommend Govee to anyone)
